app/theme.ts

```diff
@@ -7,6 +7,7 @@ const customTheme = extendTheme({
     brand: {
       backgroundDark: "#0b113e",
       backgroundMed: "#232952",
+      backgroundLight: "#b9c1c6",
       text: "#0b113e",
       textLight: "#938f8f",
       button: "#db0a40",
```

Comment:
Just a heads up that I've added new color names here to match the design system - primary, secondary, accent etc. I think we should slowly transition over to using these names in our code to avoid repetition (e.g. `backgroundDark` and `text` are the same color). I'm adding this as a task to our backlog since it will involve manual replacement of where we've used these variable names.

Reply:
Okay, I actually changed all of the backgroundDark to primary and all of the button to secondary, and removed the old variables from the theme, so that might be it for now so the task may already be completed.